cmd/cloud-run/rotate-signify-certificate/rotate_signify_certificate.py

```diff
@@ -0,0 +1,279 @@
+"""Simple PubSub handler to rotate signify certificates"""
+
+import datetime
+import os
+import base64
+import json
+import sys
+import tempfile
+import traceback
+from typing import Any, Dict, List
+import requests
+from google.cloud import secretmanager
+from flask import Flask, Response, request, make_response
+from cryptography import x509
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.serialization import pkcs7, Encoding, PrivateFormat
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+app = Flask(__name__)
+project_id: str = os.getenv("PROJECT_ID", "")
```

Comment:
consider raising an error if it's not set rather than defaulting to an empty string.